packages/contextual-menu-react/src/ContextualMenuItemLink.tsx

```diff
@@ -0,0 +1,45 @@
+import { ArrowNorthEastIcon } from "@fremtind/jkl-icons-react";
+import cn from "classnames";
+import Link from "next/link";
```

Comment:
Jeg tror det er bedre å la konsumenten velge hva slags lenke de trenger slik at denne funksjonaliteten ikke låses til brukere av Next.

Enten via `Link`/`NavLink` som nå er polymorfe etter at [denne](https://github.com/fremtind/jokul/pull/4105) ble merget eller ved å gjøre `ContextualMenuItemLink` polymorf.

Reply:
Lurt!

Reply:
Vil du at vi ser på polymorfisk løsning på dette med lenker i kontekstmenyen @olejorgenbakken ?

Reply:
Hvis dere har tid i dag hadde jeg gjerne tatt imot hjelp med det. Har et utkast som tilsynelatende fungerer, men jeg tror ikke det er så intuitivt som det kan være haha!

Har mye tid i dag, så bare å booke meg whenever i så fall 😎

Reply:
Tror de fleste av oss har en litt pakka kalender i dag med bare noen halvtimes luker her og der etter 9. Jeg sitter bak glassdøra ved kaffemaskina i 7. nå da. Ellers kan vi se på det remote i morgen eller mandag eller på kontoret tirsdag.

Reply:
Alle de fungerer for meg, men hva med remote i mandag? Fredagen er litt ymse for meg.

Reply:
Mandag etter lunch er tomt i min kalender